### PR TITLE
Refactoring the intro page

### PR DESCRIPTION
**Pull request by luzeal**
What I have tried to do is condense the page and remove information that I believe is a better fit for the FAQ page. If a comparison is still needed ("How does Qubes OS compare to XYZ"), I suggest a table rather than entire sections. The text has been copy-edited. Also, a section on : is "Qubes OS good for me (or a good fit for me)" might be a good idea. Finally, a diagram will eventually be incorporated (discussion on this [here](https://github.com/QubesOS/qubes-issues/issues/5357)).

### CODE REVIEW

**Comment by andrewdavidwong** [comment]
@marmarek and @mfc, do you have any feedback?

File: introduction/intro.md [hunk 1]
```diff
@@ -13,166 +13,30 @@ redirect_from:
...
-few days after the new attacks start to appear in the wild. By then, it's
-too late for those who have already been compromised. More advanced antivirus
-software may perform better in this regard, but it's still limited to a
-detection-based approach. New zero-day vulnerabilities are constantly being
-discovered in the common software we all use, such as our web browsers, and no
-antivirus program or firewall can prevent all of these vulnerabilities from
-being exploited.
+Qubes OS is a free and open-source security-oriented operating system meant for single-user desktop computing. 
 
+Qubes OS allows you to compartmentalize various parts of your digital life into isolated domains. If one of those compartments get compromised by a malicious email attachment carrying a ransomware, other parts of your system will remain unaffected : this is one of the key benefits of the [*security by compartmentalization*](https://www.qubes-os.org/faq/) approach taken by Qubes OS.
```
Comment (mfc) [inline introduction/intro.md]: "If one of those compartments" -> "If one of those domains"

File: introduction/intro.md [hunk 2]
```diff
@@ -13,166 +13,30 @@ redirect_from:
...
-saving disk space, to boot). Qubes even allows you to sanitize PDFs and images
-in a few clicks. Users concerned about privacy will appreciate the
-[integration][Qubes-Whonix] of [Whonix] with Qubes, which makes it easy to use
-[Tor] securely, while those concerned about physical hardware attacks will
+------------------------------------
+
+Behind the scenes, Qubes OS leverages virtualization and more specifically the [open-source Xen hypervisor](https://wiki.xen.org/wiki/Xen_Project_Software_Overview) to allow the creation and management of well-isolated virtual machines called *qubes*. Those qubes, which are also referred simply to as *domains* or *compartments*, have specific :
+
+* **Purposes** : for personal or professional projects, to manage the USB or network stack.
+* **Natures** : full-fledged or stripped-down virtual machines which can be based on Fedora, Debian or even Windows.
```
Comment (mfc) [inline introduction/intro.md]: "based on popular operating systems such as Fedora, Debian, or even Windows". this makes it more clear that there are other OS  templates as well.

**Comment by mfc** [comment]
I haven't been able to reflect on your comments of what would be a stronger approach (rather than just the readability tweaks the pull request contains). Honestly I would suggest looking at what other user-focused OSes have done for introduction: 
* https://ubuntu.com/desktop
* https://elementary.io/
* https://getfedora.org/en/workstation/
* https://tails.boum.org/about/index.en.html
* https://kubuntu.org/
* etc

graphics, screenshots, etc may be valuable.

In terms of this draft I think it looks alright, certainly simplified. probably would be good to incorporate links to the [glossary](https://www.qubes-os.org/doc/glossary/) for different terms. I think it's an incremental improvement over current version.

**Comment by andrewdavidwong** [comment]
Just waiting for any feedback from Marek on this.

File: introduction/intro.md (hunk 2 above)
Comment (luzeal) [reply introduction/intro.md]: Done

File: introduction/intro.md (hunk 1 above)
Comment (luzeal) [reply introduction/intro.md]: Done

**Comment by andrewdavidwong** [comment]
If we are hoping to merge soon, would you mind resolving the merge conflicts when you have a chance, @luzeal?

**Comment by luzeal** [comment]
@andrewdavidwong I don't mind resolving those conflicts I am just not accustomed on how to do it properly (any kind of help or support will be greatly appreciated). I have pushed a rewritten introduction page yesterday. It needs to be reviewed : see commit 459aa64.
I will create another push request to move the former content of the introduction page in the FAQ now.

File: introduction/intro.md [hunk 3]
```diff
@@ -13,30 +13,123 @@ redirect_from:
...
+          <ul>
+            <li><b>Purposes</b> : with a predefined set of one or many isolated applications, for personal or professional projects, to manage the <a href="/doc/networking/">network stack</a>, <a href="/doc/firewall/">the firewall</a>, or to fulfill other user-defined purposes.</li>
+            <li><b>Natures</b> : <a href="/doc/standalone-and-hvm/">full-fledged</a> or <a href="/getting-started/#appvms-qubes-and-templatevms">stripped-down</a> virtual machines which are based on popular operating systems such as <a href="/doc/templates/fedora">Fedora</a>, <a href="/doc/templates/debian">Debian</a> or <a href="/doc/windows/">Windows</a>.</li>
+            <li><b>Levels of trust</b> : from complete to non-existent. All windows are displayed in a unified desktop environment with <a href="https://www.qubes-os.org/getting-started/">unforgeable colored window borders</a> so different security levels are easily identifiable.</li>
+          </ul>
+      </div>
+      <div class="col-lg-9 col-md-9">
+        <h3 class="text-center add-bottom">Qubes OS Overview Example</h3>
+        <img src="/attachment/site/qubesosdiagram.png" height="600" class="center-block">
```
Comment (marmarek) [inline introduction/intro.md]: This image is missing in qubes-attachment repo.

**Comment by marmarek** [comment]
Besides one minor technicality, This looks good now, thanks!

I don't like the amount of html in markdown document, but also I don't have any better idea to have this nice layout, so it can stay as is.

**Comment by andrewdavidwong** [comment]
> I don't mind resolving those conflicts I am just not accustomed on how to do it properly (any kind of help or support will be greatly appreciated).

Well, the last time I looked at this PR a couple days ago, the "Resolve Conflicts" button was available. That would have allowed you to do it through the GUI, but now the button is grayed out because the changes are "too complex," so I think you'll have to do it through the command line. You can click on the "use the command line" link for some instructions. Here's a guide for how to resolve conflicts using the command line: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line

File: introduction/intro.md (hunk 3 above)
Comment (luzeal) [reply introduction/intro.md]: Ongoing pull request [here](https://github.com/QubesOS/qubes-attachment/pull/26)

**Comment by luzeal** [comment]
> > I don't mind resolving those conflicts I am just not accustomed on how to do it properly (any kind of help or support will be greatly appreciated).
> 
> Well, the last time I looked at this PR a couple days ago, the "Resolve Conflicts" button was available. That would have allowed you to do it through the GUI, but now the button is grayed out because the changes are "too complex," so I think you'll have to do it through the command line. [Edit: It looks like the button is available again.] You can click on the "use the command line" link for some instructions. Here's a guide for how to resolve conflicts using the command line: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line

It is actually my bad. There is a way to edit a file that is being committed to a repository trough the GUI.

One other issue is that my pull request does include two files, including an outdated version of the newer installation guide file. I am going to try to remove this extra file from the commit

**Comment by luzeal** [comment]
> > > I don't mind resolving those conflicts I am just not accustomed on how to do it properly (any kind of help or support will be greatly appreciated).
> > 
> > 
> > Well, the last time I looked at this PR a couple days ago, the "Resolve Conflicts" button was available. That would have allowed you to do it through the GUI, but now the button is grayed out because the changes are "too complex," so I think you'll have to do it through the command line. [Edit: It looks like the button is available again.] You can click on the "use the command line" link for some instructions. Here's a guide for how to resolve conflicts using the command line: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line
> 
> It is actually my bad. There is a way to edit a file that is being committed to a repository trough the GUI.
> 
> One other issue is that my pull request does include two files, including an outdated version of the newer installation guide file. I am going to try to remove this extra file from the commit

I have been unable to remove the outdated file, so I have created another cleaner pull request : https://github.com/QubesOS/qubes-doc/compare/master...luzeal:patch-4. Sorry for duplication.

**Pull request closed without merging**